src/main/java/com/vint/shop/domain/Category.java

```diff
@@ -0,0 +1,66 @@
+package com.vint.shop.domain;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Category {
```

Comment:
Please, add @Table annotation to entities

Reply:
made